ckanext/ytp/request/logic/action/delete.py

```diff
@@ -4,6 +4,7 @@
 from ckan.lib.dictization import model_dictize
 from ckan.common import c
 from sqlalchemy.sql import func
+from sqlalchemy import asc, desc
```

Comment:
```suggestion
from sqlalchemy import desc
```

- I see why you've aded `asc` here
- Buuuuut travis will fail as you're importing something which is never used
- You'll need to remove it to make travis happy